doc/news/changes/minor/20210525NicolaGiuliani

```diff
@@ -0,0 +1,3 @@
+New: Fixed a bug in clear user data for standard Triangulation.
+<br>
+(Nicola Giuliani, 2021/05/25)
```

Comment:
This file is still missing an endline at the end.

Reply:
sorry...